instat/dlgFactorLabels.vb

```diff
@@ -70,11 +69,11 @@ Public Class dlgViewFactorLabels
         ucrChkShowId.SetRDefault("TRUE")
 
         ucrChkShowPercentage.SetParameter(New RParameter("show.prc", 6))
-        ucrChkShowPercentage.SetText("Add Percentage Column")
+        ucrChkShowPercentage.SetText("Show Percentages")

```

Comment:
Okay, so I changed this to "Show Percentages", similarly so far "Show Frequencies".
This is because
1. The name is a little long otherwise
2. When adding other columns, such as an ID column, you have put "Show ID"